clarify varchar default limit and how to exceed (#5064)
fishtown-analytics/docs.getdbt.com: website/docs/reference/resource-properties/constraints.md

        data_type: date
```

> Note that by default (or when specified without a length), `varchar` in Redshift limits the maximum length of the values to 256 characters. This means that any string data exceeding 256 characters will be truncated. To allow the maximum length, use `varchar(max)`. For example, `data_type: varchar(max)`.  

Author: ```suggestion
Note that by default (or when specified without a length), `varchar` in Redshift limits the maximum length of the values to 256 characters. This means that any string data exceeding 256 characters might get truncated OR return a "value too long for character type" error. To allow the maximum length, use `varchar(max)`. For example, `data_type: varchar(max)`.  
```

> Note that by default (or when specified without a length), `varchar` in Redshift limits the maximum length of the values to 256 characters. This means that any string data exceeding 256 characters will be truncated, potentially leading to loss of information. To allow more characters, use `varchar(max)`: `data_type: varchar(max)`.  

Author: ```suggestion
Note that by default (or when specified without a length), `varchar` in Redshift limits the maximum length of the values to 256 characters. This means that any string data exceeding 256 characters will be truncated. To allow the maximum length, use `varchar(max)`. For example, `data_type: varchar(max)`.  
```

> Note that by default (or when specified without a length), `varchar` in Redshift limits the maximum length of the values to 256 characters. This means that any string data exceeding 256 characters might get truncated OR return a "value too long for character type" error. To allow the maximum length, use `varchar(max)`. For example, `data_type: varchar(max)`.  

Reviewer: Should use italics for emphasis rather than screaming case.
```suggestion
Note that Redshift limits the maximum length of the `varchar` values to 256 characters by default (or when specified without a length). This means that any string data exceeding 256 characters might get truncated _or_ return a "value too long for character type" error. To allow the maximum length, use `varchar(max)`. For example, `data_type: varchar(max)`.  
```
